packages/types/src/Bft.spec.js

```diff
@@ -36,7 +36,7 @@ describe('BftHashSignature', () => {
 
 describe('Justification', () => {
   const just = new Justification({
-    round: 32,
+    round_number: 32, // We get round_number from JSON, and change it to round
```

Comment:
I don't want to construct with `new Justification({ round_number: ... })`, would rather use either `round` or `roundNumber` (which is actually correct). So it sadly needs to cater for renames :(